collections/DOrville/MS_DOrville_153.xml

```diff
@@ -37,45 +40,132 @@
+                           <height min="140" max="155">140-155</height>
+                           <width max="80" min="60">60-80</width>
+                        </dimensions></layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="cursiva" scope="major">Current cursive script. The main text probably in one hand, the gloss and commentary perhaps in more than one other hand.</handNote>
+                     <handNote script="textualisNorthern" scope="minor">Headings in textura.</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="flourInit">No colour. Large decorative initials, in plain brown ink, for the first three books (fol. 1r, 29v, 58r); spaces left blank for most other large and small initials.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <binding notBefore="1600">
+                        <p>Sewn on four bands and bound in pasteboards covered with brown paper, with uncoloured parchment spine and brown leather corners; the spine inscribed at the top ‘Boethius | MS.’ and at the bottom ‘D’O.’</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" when="1438">1438</origDate>
+                     <origDate calendar="Gregorian" when="1438" cert="high">1438</origDate> (colophon, fol. 132v), with roughly contemporary additions
                      <origPlace>
-                        <country key="place_7000084">German</country>
+                        <country key="place_7000084">Germany (?)</country>
                      </origPlace>
                   </origin>
+                  <provenance>Perhaps from the <orgName key="org_212">Orange College of Breda</orgName> (Collegium Auriacum te Breda), whose books passed, c. 1684, to:</provenance>
+                  <provenance><orgName role="fmo" key="org_147315740">The library of the House of Orange-Nassau</orgName>: recorded in the 1686 catalogue (The Hague, KB, 78 D 14) and shelflist (of which only an 18th-century copy is known: Merseburg, Geheimes Staatsarchiv Preußischer Kulturbesitz, Oranishches Archiv, Rep. 64 R.I No. 15); included in  the 1749 auction at The Hague (see A. D. Renting, A. S. Korteweg, et al., <title>The seventeenth-century Orange-Nassau library: the catalogue compiled by Anthonie Smets in 1686, the 1749 auction catalogue, and other contemporary sources</title> (Utrecht, 1993), pp. 510–11 no. 2573); catalogue p. 266 lot 23, bought by:</provenance>
+                  <provenance><persName key="person_29618782">Pieter de Hondt (1696–1764)</persName>, printer, publisher, and bookseller of The Hague.</provenance>
```

Comment:
Do we need another value for `@role` to designate booksellers?

Reply:
the LOC codes are `bsl` bookseller and `auc` auctioneer. Happy to start using the former . I'm still a bit conflicted about the usefulness of indexing Sothebys etc.